sdk/docs/sharable/sdk/quickstart/configure/quickstart-project-structure/03-quickstart-high-level-structure.rst

```diff
@@ -1,45 +1,34 @@
 
-For simplicity Quickstart restricts itself to what is possible with
-`docker-compose`. This is sufficient for local development, debugging, and
-testing; however, for production you should consider if the dynamic resource
-management, high-availability support provided by Kubernetes is required.
+For simplicity, Quickstart restricts itself to what is possible with ``docker-compose``. 
+This is sufficient for local development, debugging, and testing; 
+however, for production you should consider if the dynamic resource management, 
+high-availability support provided by Kubernetes is required.
 
-The `LocalNet` configuration runs local versions of a super validator, the
-Canton Coin wallet application, the example application nodes, and any
-supporting services.  This allows you to run, test, and demonstrate the
-application entirely on a single machine. Please note that depending on your
-configuration, this can require considerable memory resources [5]_.
+The ``LocalNet`` configuration runs local versions of a super validator, 
+the Canton Coin wallet application, the example application nodes, and any supporting services. 
+This allows you to run, test, and demonstrate the application entirely on a single machine.
 
-Running `make setup` in the `quickstart/` directory allows you to select
-a configuration.
+Running ``make setup`` in the ``quickstart/`` directory allows you to select a configuration.
 
-Once your configuration is built and running, `make status` in `quickstart/`
-displays the associated running Docker containers.
+Once your configuration is built and running, ``make status`` in ``quickstart/`` displays the associated running Docker containers.
 
-See the Topology documentation in the `docs/` directory for more detailed
-information on the various nodes in each configuration and their
-relationship to each other.
+See the Topology documentation in the ``docs/`` directory for more detailed information on the various nodes in each configuration and their relationship to each other.
 
 Top level
 ---------
 
-Most of the top-level project directory is associated with supporting a
-portable, consistent cross-platform development environment. It does
-this using the package manager
-`Nix <https://nixos.org/download/>`__\  [6]_,
-`Direnv <https://direnv.net/>`__\  [7]_, and `Docker
-Compose <https://docs.docker.com/compose/>`__\  [8]_. The top-level setup
-ensures a consistent and repeatable dev, build, and test regardless of
-choice of environment.
+Most of the top-level project directory is associated with supporting a portable, consistent cross-platform development environment. 
+It does this using the package manager
+`Nix <https://nixos.org/download/>`__\  [5]_,
+`Direnv <https://direnv.net/>`__\  [6]_, and `Docker Compose <https://docs.docker.com/compose/>`__\  [7]_. 
+The top-level setup ensures a consistent and repeatable dev, build, and test regardless of choice of environment.
```

Comment:
Please add a couple sentences about how the user can manually install the dependencies from NIX, if NIX is not allowed in their environment.